From 5f0d8625fd178b6507d7286e57a6c3b43936f72e Mon Sep 17 00:00:00 2001
Message-ID: <5f0d8625fd178b6507d7286e57a6c3b43936f72e.1762172960.git.sam@gentoo.org>
In-Reply-To: <cc2f20ece05b98615ea9b9cb4df0a60486f810dc.1762172960.git.sam@gentoo.org>
References: <cc2f20ece05b98615ea9b9cb4df0a60486f810dc.1762172960.git.sam@gentoo.org>
From: Sam James <sam@gentoo.org>
Date: Fri, 15 Aug 2025 15:12:25 +0100
Subject: [PATCH 4/4] gas: TODO wrt enable-default-sframe vs unsupported
 targets

TODO: Where's the best place to do this check? Or maybe we ought not
to do it at all, and instead fix cfi_finish to have another conditional
s.t. it errors if the target never supports SFrames (which)?
---
 gas/configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gas/configure.ac b/gas/configure.ac
index 657dd760b84..cdaf458aafe 100644
--- a/gas/configure.ac
+++ b/gas/configure.ac
@@ -117,6 +117,7 @@ AC_ARG_ENABLE(elf_stt_common,
   yes)  ac_default_elf_stt_common=1 ;;
 esac])dnl
 
+# TODO: Forbid this on unsupported targets
 ac_default_sframe=unset
 # Provide a configure time option to override our default.
 AC_ARG_ENABLE(default-sframe,
-- 
2.51.2

